### ISSUE TITLE
Valuation Method LIFO missing in Documentation :: 

### ISSUE THREAD

**Issue by frewper** [frappe/erpnext#34815]
### Information about bug

This is a documentation related bug. I have checked the documentation site and did not find a place to report the documentation related error. 

There are 3 types of valuation methods in ERPNext. 

1. FIFO
2. LIFO
3. Average Moving 

LIFO is not mentioned in the documentation link below. Need to add the same. 
https://docs.erpnext.com/docs/v14/user/manual/en/stock/articles/item-valuation-transactions

Their is a link to the following blog post as well, which is referred in the documentation and needs to be updated. 
https://frappe.io/blog/erpnext-features/inventory-valuation-method-fifo-vs-moving-average

### Module

stock

### Version

ERPNext: v14.17.2 (version-14)
Frappe Framework: v14.26.2 (version-14)

### Installation method

manual install

### Relevant log output / Stack trace / Full Error Message.

```shell
Not Applicable
```

**Comment by zaid2229** [comment]
Please update documentation for LIFO  valuation method.

**Comment by Nihantra-Patel** [comment]
https://github.com/frappe/erpnext/pull/29296

- [FIFO and Moving Average calculation difference](https://docs.erpnext.com/docs/user/manual/en/calculation-of-valuation-rate-in-fifo-and-moving-average)


In LIFO valuation method, the most recently acquired inventory items are the first to be sold or used. This means that the cost of goods sold (COGS) is based on the cost of the most recent purchases, and the ending inventory consists of the earliest purchases.

**Example:**

Consider the same transactions as before:

Date | Transaction | Qty | Unit Cost
-----|-------------|-----|----------
1-4-2020 | Purchase | 10 | 100
6-4-2020 | Purchase | 20 | 120
10-4-2020 | Sale | 15 | ?

**Calculating Valuation Rate at the time of sale:**

**As per LIFO:**

Since this is LIFO, we will consume quantities from the most recent transactions first. Therefore, to make a sale of 15 qty, we will take all 15 qty from the second transaction (purchase on 6-4-2020).

The valuation rate at the time of sale would be:

15 * 120 = 1800

This leaves us with 15 qty in stock at a cost of 100 (from the first transaction) amounting to 1500.

So, even though the quantity sold is the same, the stock value is different under LIFO compared to FIFO and Moving Average, but all three methods amount to a total of 3400.

In summary, while FIFO (First In, First Out) and LIFO (Last In, First Out) valuation methods differ in which inventory items are deemed sold first, both aim to calculate the cost of goods sold and ending inventory accurately.
